api/src/api/agencies_v1/agency_schema.py

```diff
@@ -0,0 +1,77 @@
+from src.api.schemas.extension import Schema, fields
+from src.api.schemas.response_schema import AbstractResponseSchema, PaginationMixinSchema
+from src.pagination.pagination_schema import generate_pagination_schema
+
+
+class AgencyFilterV1Schema(Schema):
+    agency_id = fields.Integer()
+
+
+class AgencyListRequestSchema(Schema):
+    filters = fields.Nested(AgencyFilterV1Schema())
+    pagination = fields.Nested(
+        generate_pagination_schema(
+            "AgencyPaginationV1Schema",
+            ["created_at"],
+        ),
+        required=True,
+    )
+
+
+class AgencyContactInfoSchema(Schema):
+    """Schema for agency contact information"""
+
+    contact_name = fields.String()
+    address_line_1 = fields.String()
+    address_line_2 = fields.String(allow_none=True)
+    city = fields.String()
+    state = fields.String()
+    zip_code = fields.String()
+    phone_number = fields.String()
+    primary_email = fields.String()
+    secondary_email = fields.String(allow_none=True)
+
+
+class AgencyResponseSchema(Schema):
+    """Schema for agency response"""
+
+    agency_id = fields.Integer()
+    agency_name = fields.String()
+    agency_code = fields.String()
+    sub_agency_code = fields.String(allow_none=True)
+    assistance_listing_number = fields.String()
+    agency_submission_notification_setting = fields.String()  # Enum value
+
+    # Agency contact info as nested object
+    agency_contact_info = fields.Nested(AgencyContactInfoSchema, allow_none=True)
+
+    # Boolean flags
+    is_test_agency = fields.Boolean()
```

Comment:
We probably want to filter out any test agencies (like how we filter draft opportunities), the legacy system does that to some degree. The agency transform job already handles setting this flag for known test agencies

Reply:
Added a filter to this and removed from the schema since this will always be `false`